src/testing/jest/test/jest-config.spec.ts

```diff
@@ -6,12 +6,12 @@ import path from 'path';
 
 describe('jest-config', () => {
   it('pass --maxWorkers=2 arg when --max-workers=2', () => {
-    const args = ['test', '--ci', '--e2e', '--max-workers=2'];
+    const args = ['test', '--ci', '--max-workers=2'];
```

Comment:
This test was exactly the same as the one below it. I'm guessing this is copy/pasta, but could be wrong. The test name differentiation suggests this shouldn't run with the `--e2e` flag.  I can pull this into a separate commit if desired